From 084de4774a4145381b7a1d8a16f01d8d703a6498 Mon Sep 17 00:00:00 2001
From: Ye Li <ye.li@nxp.com>
Date: Thu, 8 Jul 2021 01:36:21 -0700
Subject: [PATCH 02/19] LFU-188-2 imx8mp_evk: Remove reservation for MCU RPMSG
 buffer

Since u-boot has moved to top DDR not in the DDR memory before optee.
It does not need to reserve this 16M buffer used by RPMSG.

Signed-off-by: Ye Li <ye.li@nxp.com>
Reviewed-by: Peng Fan <peng.fan@nxp.com>
---
 board/freescale/imx8mp_evk/imx8mp_evk.c | 11 -----------
 1 file changed, 11 deletions(-)

diff --git a/board/freescale/imx8mp_evk/imx8mp_evk.c b/board/freescale/imx8mp_evk/imx8mp_evk.c
index c4997439cc..eda2fe4a85 100644
--- a/board/freescale/imx8mp_evk/imx8mp_evk.c
+++ b/board/freescale/imx8mp_evk/imx8mp_evk.c
@@ -486,17 +486,6 @@ int board_late_init(void)
 	return 0;
 }
 
-#ifdef CONFIG_IMX_BOOTAUX
-ulong board_get_usable_ram_top(ulong total_size)
-{
-	/* Reserve 16M memory used by M core vring/buffer, which begins at 16MB before optee */
-	if (rom_pointer[1])
-		return gd->ram_top - SZ_16M;
-
-	return gd->ram_top;
-}
-#endif
-
 #ifdef CONFIG_ANDROID_SUPPORT
 bool is_power_key_pressed(void) {
 	return (bool)(!!(readl(SNVS_HPSR) & (0x1 << 6)));
-- 
2.33.0

